lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -45,43 +45,29 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
    * @return {!Element}
    */
   _renderOpportunity(audit, index, scale) {
-    const tmpl = this.dom.cloneTemplate('#tmpl-lh-opportunity', this.templateContext);
-    const element = this.dom.find('.lh-load-opportunity', tmpl);
-    element.classList.add(`lh-load-opportunity--${Util.calculateRating(audit.result.score)}`);
+    const oppTmpl = this.dom.cloneTemplate('#tmpl-lh-opportunity', this.templateContext);
+    const element = this.populateAuditValues(audit, index, oppTmpl);
+    element.classList.add(`lh-audit--${Util.calculateRating(audit.result.score)}`);
```

Comment:
looks like this is now already set in `populateAuditValues` by `_setRatingClass()`

Reply:
lol `populateAuditValues` also has a stray `auditEl.classList.add(`lh-audit--${audit.result.scoreDisplayMode}`);` line right before calling `_setRatingClass`

Reply:
hah. thanks for catching both of these